### PR TITLE
New package: wtype-0.3

### PR DESCRIPTION
**Pull request by Eluminae**

### CODE REVIEW

File: srcpkgs/wtype/template [hunk 1]
```diff
@@ -0,0 +1,19 @@
+# Template file for 'wtype'
+pkgname=wtype
+version=0.3
+revision=1
+archs="~*-musl"
+build_style=meson
+hostmakedepends="pkg-config wayland-devel"
+makedepends="libxkbcommon-devel wayland-devel"
+depends="libxkbcommon wayland"
```
Comment (pullmoll) [inline srcpkgs/wtype/template:9]: This line is not required. The dependencies should be detected automatically.

**Review by pullmoll — CHANGES_REQUESTED** [review]

File: srcpkgs/wtype/template (hunk 1 above)
Comment (Eluminae) [reply srcpkgs/wtype/template]: okay I removed it

**Comment by mcbloch** [comment]
Tried on `Linux 5.9.16_1 #1 SMP 1608571490 x86_64 GNU/Linux` and works.

**Comment by Eluminae** [comment]
I disabled musl package but next release will be musl compatible https://github.com/atx/wtype/issues/20

**Comment by the-maldridge** [comment]
@Eluminae can you pull in the patch that was applied upstream so that this release works on musl as well?

**Comment by Eluminae** [comment]
> @Eluminae can you pull in the patch that was applied upstream so that this release works on musl as well?

Good idea. This is done :)

**Comment by ajshell1** [comment]
My apologies. Closing.

**Comment by Eluminae** [comment]
hello there ?

**Comment by ericonr** [comment]
Can you please rebase so we get a CI run?

**Comment by Eluminae** [comment]
> And how does this compare to ydotool?

I know nothing about ydotool

**Comment by ericonr** [comment]
It doesn't work on wayfire, investigating who to blame for it.

**Comment by ericonr** [comment]
It works on sway.

**Pull request merged**